packages/driver/cypress/integration/commands/actions/selectFile_spec.js

```diff
@@ -54,6 +54,7 @@ describe('src/cy/commands/actions/selectFile', () => {
           fileName: 'bar.json',
         },
         Cypress.Buffer.from('baz'),
+        Uint8Array.from([98, 97, 122]),
```

Comment:
Would it be good to drop a comment here that this is `baz` in ASCII? It took me a minute to figure that out and now the rest of the test makes sense!

Reply:
Will do.